jobs/ingestor_cloudwatch/templates/config/input_and_output.conf.erb

```diff
@@ -12,6 +12,22 @@ input
   }
 }
 
+filter
+{
+    mutate {
+        rename => {"[cloudwatch_logs][tags][environment]"=>"environment"}
+        rename => {"[cloudwatch_logs][tags][OrganizationGUID]"=>"[@cf][org_id]"}
+        rename => {"[cloudwatch_logs][tags][SpaceGUID]"=>"[@cf][space_id]"}
+        rename => {"[cloudwatch_logs][tags][Organizationname]"=>"[@cf][org]"}
+        rename => {"[cloudwatch_logs][tags][Createdat]"=>"Created_at"}
```

Comment:
I think maybe we should just drop this field on ingestion. the `CreatedAt` tag on the RDS instance is the time that the database was created. When we bring it onto the Cloudwatch log group, the original meaning gets lost and the meaning of this field becomes confusing. 

There's also an `Updatedat` tag that could appear that we should drop

Reply:
added in